test/gscraper_web/views/page_view_test.exs

```diff
@@ -0,0 +1,3 @@
+defmodule GscraperWeb.PageViewTest do
```

Comment:
This test can be removed too because there is no test case.

Reply:
Removed in 182a352.